consensus/src/runway/packer.rs

```diff
@@ -133,25 +132,30 @@ mod tests {
             keychain,
             SESSION_ID,
         );
-        let (exit_tx, exit_rx) = oneshot::channel();
+        let (_, exit_rx) = oneshot::channel();
```

Comment:
You need to keep the sender, otherwise everything immediately closes. <_<

There is a crucial and unintuitive difference between `_` and `_variable_name`.

Reply:
Rather important, thanks for catching this!

Reply:
Rather important, thanks for catching this!